assets/js/form.js

```diff
@@ -0,0 +1,72 @@
+function startForm(containingClass, id) {
+  $('.' + containingClass).append("<form class='cmxform' id=" + id + " method='get' action='' autocomplete='off'>");
+}
+
+function endForm(containingClass, id) {
+  $('.' + containingClass).append("</form>");
+}
+
+function submitButton(containingClass) {
+  $("." + containingClass + " form").append("<input type='submit' value='Submit' class='submit vita-form-button background-primary'>");
+}
+
+function newFormTitle(containingClass, title) {
+  $("." + containingClass + " form").append("<h1 class='vita-form-title'>" + title + "</h1>");
+}
+
+function newSubheading(containingClass, subheading) {
+  $("." + containingClass + " form").append("<h2 class='vita-form-subheading'>" + subheading + "</h2>");
+}
+
+function newTextField(containingClass, id, label, required) {
+  if (required == true) {
```

Comment:
Is there any way that we could make this so there's not so much duplicate code? Because the only part that changes in this if/else statement is the class='vita-form-required'.. Could we not just do the part before this, and then do an if(required) { append that part }
finish it off?

If it hinders readability too much though, what you have now looks good